server/budgettracker.business/Api/Contracts/BudgetApi/UpdateBudget/UpdateBudgetArgumentApiContract.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace budgettracker.business.Api.Contracts.BudgetApi.UpdateBudget
+{
+    public class UpdateBudgetArgumentApiContract : IApiContract
+    {
+        [JsonProperty("budget-value")]
+        public UpdateBudgetRequestContract BudgetValue { get; set; }
```

Comment:
Normally I wouldn't care about this to request a change but since there are other things you're changing, you might as well do this... I called these values... plural. The UpdateBudgetRequestContract contains multiple values... not just one. So call this budget-values and BudgetValues.

Reply:
Are you going to rename this?